memento.root/usr/share/clearwater/cassandra-schemas/memento.sh

```diff
@@ -1,8 +1,119 @@
-#! /bin/bash
-if [[ ! -e /var/lib/cassandra/data/memento ]];
-then
-    echo "CREATE KEYSPACE memento WITH strategy_class='org.apache.cassandra.locator.SimpleStrategy' AND strategy_options:replication_factor=2;
-          USE memento;
-          CREATE TABLE call_lists (impu text PRIMARY KEY, dummy text) WITH read_repair_chance = 1.0;
-    " | cqlsh -2
+#!/bin/bash
+
+keyspace=$(basename $0|sed -e 's#^\(.*\)[.]sh$#\1#')
```

Comment:
The GR code at the bottom of the script is generic and copy-pasted into similar files in other repositories.